lib/matplotlib/tests/test_patches.py

```diff
@@ -197,7 +198,7 @@ def test_patch_alpha_override():
     ax.set_ylim([-1, 2])
 
 
-@pytest.mark.style('default')
+@mpl.style.context('default')
```

Comment:
There's an `mstyle` import here already, that you either could use, or drop and change the other usage to match.

Reply:
mpl.style it is, then.